src/controllers/payment.controller.js

```diff
@@ -0,0 +1,99 @@
+import {
+  stripeToken,
+  paymentMethod,
+  charge,
+  stripeListener,
+  sendEmails,
+} from '../utils';
+import { orderService } from '../services';
+import { stripe } from '../helpers';
+
+const { ADMIN_EMAIL } = process.env;
+const makePayment = async (req, res) => {
```

Comment:
in this function, if an error occurs, it is not caught, and the response is not handled appropriately. Instead of throwing an error, it is better to return a meaningful error message to the client.